### ISSUE TITLE
Malignant hyperthermia due to exercise

### ISSUE THREAD

**Issue by kanems** [obophenotype/human-phenotype-ontology#9729]
**Preferred term label:**
Malignant hyperthermia due to exercise

**Synonyms**
Exercise-induced malignant hyperthermia
Exertional heat stroke

**Definition (free text, please give PubMed ID)**
Malignant hyperthermia that is characterized by exercise-induced life-threatening hyperthermia with a body temperature over 40°C and signs of encephalopathy ranging from confusion to convulsions or coma. Incidence increases with rising ambient temperature and relative humidity. Manifestations may include rhabdomyolysis (presenting with myalgia, muscle weakness, and myoglobinuria), tachycardia, and in severe cases multiorgan failure.
https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=24151&disease=Exercise-induced-malignant-hyperthermia&search=Disease_Search_Simple 

**Parent term (use hpo.jax.org/app)**
Malignant hyperthermia HP:0002047

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:466650

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034732
